### DISCUSSION TITLE
Problem with install pg manually

### DISCUSSION THREAD

**Discussion by dihreyaraujo** [vercel/community#6013] [Help]
### Summary

Good evening, I'm having a problem with a backend project of mine at Vercel. I'm deploying a Github repository on Vercel and the build goes normally, but when I look at the Runtime Log, it gives the following error: Please install pg package manually
    at ConnectionManager._loadDialectModule (/var/task/node_modules/sequelize/lib/dialects/abstract/connection-manager.js:55:15).

I don't know what it could be, as my repository already has the "pg" package as a project dependency. All other dependencies are being installed normally, I believe, but "pg" is not. This means that I am unable to make request calls and use the API as I would like.

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/99283526/158a1c86-c75f-4c10-81ea-d9dbc1e67bed)

![image](https://github.com/vercel/community/assets/99283526/2a6c7dce-e118-481a-9c34-0c5c7f470c08)

![image](https://github.com/vercel/community/assets/99283526/ea0cd894-76c0-47b1-85ec-4c8362f651b6)

**Comment by alinabezux** [comment]
Have you fixed this issue?

**Reply by versecafe** [reply]
@alinabezux there seems to be a fix here https://github.com/sequelize/sequelize/issues/6907#issuecomment-2028971121 but this has been a long standing issue as the thread goes back to 2016, but the most recent addition seems to have a fix and there are several other solutions provided, hopefully one works for you.
